doc/source/contributing.rst

```diff
@@ -575,7 +575,7 @@ the `flake8 <https://pypi.org/project/flake8>`_ tool
 and report any stylistic errors in your code. Therefore, it is helpful before
 submitting code to run the check yourself on the diff::
 
-   git diff master -u -- "*.py" | flake8 --diff
+   git diff upstream/master -u -- "pandas/*.py" | flake8 --diff
```

Comment:
why `"pandas/*.py"`? I think we should also check `.py` files in `scripts/`, `ci/` and everywhere else.

Reply:
@datapythonista 
Happy to remove that too, if desired. As the PR stands, this was another inconsistency, and all the other commands (i.a. also the raison-d'être for `grep`) are filtering to `pandas/`.

Reply:
Don't think it's relevant here but want to call out that there can certainly be performance implications to not restricting queries to the `pandas` directory. For instance, when grepping from the top level if you don't restrict to that folder you could have a command that runs potentially much longer if it starts searching through the `asv_bench` environments

Reply:
True, but it's only checking the diff anyway, which - even in the `asv_bench` - will be relatively small (or, for most PRs, non-existent)